po/fr.po

```diff
@@ -25129,1346 +25075,1362 @@ msgstr "Exécution en silence"
+msgstr "Ajoute un serveur distant"
 
 #: /tmp/fish/implicit/share/completions/git.fish:43
-#, fuzzy
 msgid "Removes a remote"
-msgstr "Supprimer une clé"
+msgstr "Supprime un serveur distant"
 
 #: /tmp/fish/implicit/share/completions/git.fish:44
-#, fuzzy
 msgid "Shows a remote"
-msgstr "Afficher les entrées ARP"
+msgstr "Affiche un serveur distant"
 
 #: /tmp/fish/implicit/share/completions/git.fish:45
 msgid "Deletes all stale tracking branches"
-msgstr ""
+msgstr "Supprime toutes les branches dépassées suivies"
 
 #: /tmp/fish/implicit/share/completions/git.fish:46
-#, fuzzy
 msgid "Fetches updates"
-msgstr "Récupérer les paquets sources"
+msgstr "Rapatrie les mises à jour"
 
 #: /tmp/fish/implicit/share/completions/git.fish:47
-#, fuzzy
 msgid "Renames a remote"
-msgstr "Renommer un disque"
+msgstr "Renomme un serveur distant"
 
 #: /tmp/fish/implicit/share/completions/git.fish:48
 msgid "Sets the default branch for a remote"
-msgstr ""
+msgstr "Paramètre une branche par défaut pour un serveur distant"
 
 #: /tmp/fish/implicit/share/completions/git.fish:49
 msgid "Changes URLs for a remote"
-msgstr ""
+msgstr "Change les URL pour un serveur distant"
 
 #: /tmp/fish/implicit/share/completions/git.fish:50
+#, fuzzy
 msgid "Changes the list of branches tracked by a remote"
-msgstr ""
+msgstr "Change la liste des branches suivies par un serveur distant"
 
 #: /tmp/fish/implicit/share/completions/git.fish:51
 msgid "Once the remote information is set up git fetch <name> is run"
 msgstr ""
+"Une fois l’information distante configurée, git fetch <nom> est exécuté"
 
 #: /tmp/fish/implicit/share/completions/git.fish:52
 msgid "Import every tag from a remote with git fetch <name>"
 msgstr ""
+"Importer toutes les étiquettes d’un serveur distant avec git fetch <nom>"
 
 #: /tmp/fish/implicit/share/completions/git.fish:53
 msgid "Don't import tags from a remote with git fetch <name>"
-msgstr ""
+msgstr "Ne pas importer d’étiquettes d’un serveur distant avec git fetch <nom>"
 
 #: /tmp/fish/implicit/share/completions/git.fish:54
 msgid "Add to the list of currently tracked branches instead of replacing it"
-msgstr ""
+msgstr "Ajouter à la liste des branches suivies au lieu de la remplacer"
 
 #: /tmp/fish/implicit/share/completions/git.fish:55
 msgid "Manipulate push URLs instead of fetch URLs"
-msgstr ""
+msgstr "Manipuler les URL de poussée au lieu de celles de rapatriement"
 
 #: /tmp/fish/implicit/share/completions/git.fish:56
 msgid "Add new URL instead of changing the existing URLs"
-msgstr ""
+msgstr "Ajouter une nouvelle URL au lieu de changer celle en place"
 
 #: /tmp/fish/implicit/share/completions/git.fish:57
 msgid "Remove URLs that match specified URL"
-msgstr ""
+msgstr "Supprimer les URL correspondant à celle spécifiée"
 
 #: /tmp/fish/implicit/share/completions/git.fish:58
 msgid "Remote heads are not queried, cached information is used instead"
 msgstr ""
+"Les HEAD distantes ne sont pas requises, l’information en cache est utilisée "
```

Comment:
“requises” ≠︎ “queried”; use “examinées” instead?

Reply:
*Requises* is a conjugation of *requérir*, which means *to request*. There is also *requêter*, derived from *requête*, and which means *to query*, but it's an anglicism used for databases; on the other hand, *queried* seems to be used here to say *requested*, not *queried using a DB query*, and *requérir* is fairly common, so I judge it to be a better translation.

Regarding *examiner*, it suggests an fairly intrusive test, or a judgement, not a simple test, and does not seem to fit either.

Reply:
Ah, I’m not familiar with the technical use of _requérir_, hence my reading of _requises_ as a (female form) of the adjective _mandatory_. Let’s leave it you suggested, then.